pkg/services/ngalert/api/api_ruler.go

```diff
@@ -526,6 +504,22 @@ func (srv RulerSrv) updateAlertRulesInGroup(c *models.ReqContext, namespace *mod
 	return response.JSON(http.StatusAccepted, util.DynMap{"message": "rule group updated successfully"})
 }
 
+func toGettableRuleGroupConfig(groupName string, rules []*ngmodels.AlertRule, namespaceID int64, provenanceRecords map[string]ngmodels.Provenance) apimodels.GettableRuleGroupConfig {
+	ruleNodes := make([]apimodels.GettableExtendedRuleNode, 0, len(rules))
+	var interval time.Duration
+	if len(rules) > 0 {
+		interval = time.Duration(rules[0].IntervalSeconds) * time.Second
+	}
```

Comment:
we use two different approaches :) in two different methods: one uses the interval of the first rule in the group (sorted by ID by query) and another uses the interval of the last rule. 
As long as those intervals must be the same, it really does not matter what element we take